jupyter-book/tuning/parameter_tuning_module_take_away.md

```diff
@@ -9,10 +9,10 @@
 - The search for the best hyperparameters can be automated with a grid-search
   approach or a randomized search approach;
 - A grid-search is expensive and does not scale when the number of
-  hyperparameters to optimize increase. Besides, the combination are sampled
+  hyperparameters to optimize increase. Additionally, the combination are sampled
   only on a regular grid.
```

Comment:
This whole paragraph can be improved to avoid a possible confusion with mathematical scaling. How about something in the line of:
>- A grid-search can be computationally expensive and becomes less attractive as
  the number of hyperparameters to explore increases. Moreover, the combinations
  are sampled on a fixed, regular grid.